context/app/static/assets/svg/workspaces.svg

```diff
@@ -0,0 +1 @@
+<svg enable-background="new 0 0 24 24" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m0 0h24v24h-24z" fill="none"/><path d="m6 13c-2.2 0-4 1.8-4 4s1.8 4 4 4 4-1.8 4-4-1.8-4-4-4zm6-10c-2.2 0-4 1.8-4 4s1.8 4 4 4 4-1.8 4-4-1.8-4-4-4zm6 10c-2.2 0-4 1.8-4 4s1.8 4 4 4 4-1.8 4-4-1.8-4-4-4z"/></svg>
```

Comment:
@mccalluc The three circle icon is the 'workspace' icon that Material Design uses for workspaces. We could always use something else, but if they have a predefined icon, I'll typically use it since it could be what users are used to seeing.